src/hooks/usePayByLinkFilters.ts

```diff
@@ -0,0 +1,49 @@
+import { useConfigContext } from '../core/ConfigContext';
+import { useFetch } from './useFetch';
+import { EMPTY_OBJECT } from '../utils';
+import { useMemo } from 'preact/hooks';
+
+const usePayByLinkFilters = (enabled?: boolean) => {
+    const { payByLinkFilters: getPayByLinkFiltersEndpointCall } = useConfigContext().endpoints;
```

Comment:
Can you get the getStoresEndpointCall from here like `const { payByLinkFilters: getPayByLinkFiltersEndpointCall, getStores: getStoresEndpointCall } = useConfigContext().endpoints;`?